lib/peep.rb

```diff
@@ -0,0 +1,32 @@
+require 'pg'
+
+class Peep
+  attr_reader :time, :message, :id
+
+  def initialize(time:, message:, id:)
```

Comment:
does having the colon after each parameter, i..e time:, change the functionality of that at all?